include/opc/ua/protocol/utils.h

```diff
@@ -19,6 +19,9 @@
 
 #pragma once
 
+#ifdef WIN32
+#include <algorithm>
+#endif
```

Comment:
I just didn't know how will it affect Linux build, so made it applicable only for Windows.